packages/presets/gql-tag-operations/tests/gql-tag-operations.spec.ts

```diff
@@ -1,5 +1,8 @@
 import { executeCodegen } from '@graphql-codegen/cli';
+import { mergeOutputs } from '@graphql-codegen/plugin-helpers';
 import '@graphql-codegen/testing';
+import { validateTs } from '@graphql-codegen/testing';
+import { readFile } from 'fs/promises';
```

Comment:
This is not available in Node.js 12 😭

We want to drop support soon, but for now we have to resolve this failing CI

Reply:
oh... 🙈 
I fixed it! 53ef642